app/src/main/java/org/sopt/sample/presentation/ui/main/gallery/GalleryFragment.kt

```diff
@@ -0,0 +1,19 @@
+package org.sopt.sample.presentation.ui.main.gallery
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.sopt.sample.R
+import org.sopt.sample.databinding.FragmentGalleryBinding
+import org.sopt.sample.presentation.common.base.BindingFragment
+
+class GalleryFragment : BindingFragment<FragmentGalleryBinding>(R.layout.fragment_gallery) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+    }
+
+}
```

Comment:
```suggestion
class GalleryFragment : BindingFragment<FragmentGalleryBinding>(R.layout.fragment_gallery)
```

onViewCreated에서 별도 작업이 필요하지 않다면 이렇게 정리해봐도 좋을 것 같아요~ SearchFragment도 마찬가지입니당!